app-modules/audit/composer.json

```diff
@@ -6,9 +6,7 @@
   "license": "proprietary",
   "require": {
     "filament/filament": "^3.0",
-    "owen-it/laravel-auditing": "dev-feature/queued-auditing@dev",
-    "spatie/laravel-settings": "^3.2",
-    "filament/spatie-laravel-settings-plugin": "^3.0"
```

Comment:
It's fine that we moved these two to be dependencies in the root `composer.json` because we are using them in the root Laravel setup now as well.

But we may want to leave them here too. To keep continuity that this Module requires these items.

What do you think @dgoetzit and @danharrin?

Reply:
Although we probably have missed a couple of places where this is the case, I think we should strive to add/keep dependencies where they would be necessary if the modules were standalone.

Reply:
I agree, though it is annoying that we will probably have to come into here and ensure that these numbers are kept up-to-date with root. I think it is important we keep track of the packages a module depends upon to some degree.

So @danharrin if you can just add these back here as well. You can keep them in root too though.